airbyte-integrations/connectors/source-railz-ai/source_railz_ai/components.py

```diff
@@ -0,0 +1,370 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import time
+from dataclasses import InitVar, dataclass, field
+from datetime import datetime
+from typing import Any, Iterable, Mapping, Optional, Union, List
+
+import requests
+from airbyte_cdk.models import AirbyteMessage, SyncMode, Type
+from airbyte_cdk.sources.declarative.auth.token import AbstractHeaderAuthenticator
+from airbyte_cdk.sources.declarative.exceptions import ReadException
+from airbyte_cdk.sources.declarative.datetime.min_max_datetime import MinMaxDatetime
+from airbyte_cdk.sources.declarative.interpolation.interpolated_string import InterpolatedString
+from airbyte_cdk.sources.declarative.requesters.request_option import RequestOption
+from airbyte_cdk.sources.declarative.retrievers import SimpleRetriever
+from airbyte_cdk.sources.declarative.extractors import RecordSelector
+from airbyte_cdk.sources.declarative.stream_slicers import DatetimeStreamSlicer, SingleSlice
+from airbyte_cdk.sources.declarative.types import Config, Record, StreamSlice, StreamState
+from airbyte_cdk.sources.streams.core import Stream, StreamData
+from airbyte_cdk.sources.streams.http.requests_native_auth import BasicHttpAuthenticator, TokenAuthenticator
+from dataclasses_jsonschema import JsonSchemaMixin
+from requests.adapters import HTTPAdapter, Retry
+
+
+@dataclass
+class AccessTokenAuthenticator(AbstractHeaderAuthenticator, JsonSchemaMixin):
```

Comment:
@Allexik for each custom component, can you please: 

1. Add in the docstring why it's needed / why the current CDK components are not sufficient
2. Create a GH issue requesting that it's added to the lowcode CDK
3. link the issue in the docstring

Reply:
@sherifnada thanks for review. @Allexik offboarded from GL. @grubberr will be responsible for this PR. 

@grubberr can you take a look?